views/trust-involved.html

```diff
@@ -0,0 +1,71 @@
+{% extends "layout.html" %}
+
+{% set title = "Individuals or entities involved in the trust" %}
+
+{% block pageTitle %}
+  {% include "includes/page-title.html" %}
+{% endblock %}
+
+{% block backLink %}
+  {% include "includes/back-link.html" %}
+{% endblock %}
+
+{% block content %}
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+      <span class="govuk-caption-l">{{ pageData.trustName }}</span>
+      <h1 class="govuk-heading-xl">{{ pageParams.title }}</h1>
+
+      <form class="form" method="post">
+
+        {% set fieldParam = {
+          name: 'typeOfTrustee',
+          label: 'Which type of individual or entity do you want to add to this trust?',
+          error: errors.typeOfTrustee,
+          items: [
+            {
+              value: "historical",
```

Comment:
in controller pass to template (render method)
```
return res.render(
      templateName,
      {
        backLinkUrl: `${config.TRUST_DETAILS_URL}/${req.params['id']}`,
        templateName,
        pageData: {
            trusteeType: TrusteeType(Choice),
        },
        pageParams: {
        ...
```

in template
```        
{% set fieldParam = {
          name: 'typeOfTrustee',
          label: 'Which type of individual or entity do you want to add to this trust?',
          error: errors.typeOfTrustee,
          items: [
            {
              value: pageData.trusteeType.HISTORICAL,
              ...
              ```

Reply:
We are now passing in the TrusteeType via the controller and accessing in HTML. 

**trust.involved.controller.ts**

`return res.render(
      templateName,
      {
        backLinkUrl: `${config.TRUST_DETAILS_URL}/${req.params['id']}`,
        templateName,
        pageData: {
          trusteeType: TrusteeType,
        },
        pageParams: {
          title: TRUST_INVOLVED_TEXTS.title,
          checkYourAnswersUrl: config.CHECK_YOUR_ANSWERS_URL,
        },`

**trust-involved.html**

`              value: pageData.trusteeType.HISTORICAL,
              text: "Former beneficial owner",
              hint: {
                text: "This is any individual or legal entity that was previously a registrable beneficial owner of the overseas entity, due to being a trustee of this trust."
              }`